alicloud/table_alicloud_oss_bucket.go

```diff
@@ -276,11 +282,11 @@ func bucketSSEConfiguration(_ context.Context, d *transform.TransformData) (inte
 	}, nil
 }
 
-func bucketAka(ctx context.Context, d *transform.TransformData) (interface{}, error) {
-	plugin.Logger(ctx).Trace("getBucketAka")
+func bucketARN(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("bucketARN")
 	bucket := d.HydrateItem.(oss.BucketProperties)
 
-	return []string{"acs:oss:::" + bucket.Name}, nil
+	return "acs:oss:::" + bucket.Name, nil
```

Comment:
@rajeshbal65  Can we update the `arn` format to `arn:acs:oss:cn-hangzhou:123456789xxxxxxx:bucket/my-bucket-name-xxx`
https://www.alibabacloud.com/help/doc-detail/172061.htm#title-7v2-mxj-6lb